skipper.go

```diff
@@ -53,8 +53,14 @@ type Options struct {
 	// Skip TLS certificate check for etcd connections.
 	EtcdInsecure bool
 
-	// Kubernetes API base URL for ingress. If not specififed,
-	// Kubernetes ingress is disabled.
+	// If set enables skipper to generate based on ingress resources in kubernetes cluster
+	Kubernetes bool
+
+	// If set makes skipper to authenticate with the kubernetes API server with service account assigned to the skipper POD
+	// If omitted skipper will rely on kubectl proxy to authenticate with API server
+	KubernetesInCluster bool
+
+	// Kubernetes API base URL. Only makes sense if KubernetesInCluster is set to false. If omitted and skipper not deployed to the cluster, the default API URL will be used
```

Comment:
nitpick: "If omitted and skipper is not deployed to the cluster, .." (missing "is")